### PR TITLE
chore: add changeset of account api

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add changeset of Account API.

## Testing

N/A

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: .changeset/warm-pugs-divide.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+'@logto/schemas': minor
+'@logto/connector-kit': minor
+---
+
+add account api
+
+Introduce the new Account API, designed to give end users direct API access without needing to go through the Management API, here is the highlights:
+
+1. Direct access: The Account API empowers end users to directly access and manage their own account profile without requiring the relay of Management API.
+2. User profile and identities management: Users can fully manage their profiles and security settings, including the ability to update identity information like email, phone, and password, as well as manage social link connections. May also support MFA and SSO in the future.
```
Comment (gao-sun) [inline .changeset/warm-pugs-divide.md:14]: "may also" sounds weird. maybe "MFA and ... will be added ..."

File: .changeset/warm-pugs-divide.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+3. Global access control: Admin has full, global control over access settings, can customize each fields.
+4. Seamless authorization: Authorizing is easier than ever! Simply use client.getAccessToken() to obtain an access token and attach it to the Authorization header.
```
Comment (gao-sun) [inline .changeset/warm-pugs-divide.md:16]: this sounds confusing. better to change to a more clear version

Comment (wangsijie) [reply .changeset/warm-pugs-divide.md]: updated

File: .changeset/warm-pugs-divide.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+'@logto/schemas': minor
+'@logto/connector-kit': minor
+---
+
+add account api
+
+Introduce the new Account API, designed to give end users direct API access without needing to go through the Management API, here is the highlights:
+
+1. Direct access: The Account API empowers end users to directly access and manage their own account profile without requiring the relay of Management API.
+2. User profile and identities management: Users can fully manage their profiles and security settings, including the ability to update identity information like email, phone, and password, as well as manage social connections. MFA and SSO support is coming soon.
```
Comment (gao-sun) [inline .changeset/warm-pugs-divide.md:14]: are coming soon

**Review by gao-sun — APPROVED** [review]

**Pull request merged**